commands.go

```diff
@@ -271,6 +271,13 @@ func (c *cmdable) Ping() *StatusCmd {
 	return cmd
 }
 
+func (c *cmdable) Wait(n_slaves uint, n_seconds uint) *IntCmd {
```

Comment:
Go convention is to use camelCased var names. So it should be `numSlaves, numSeconds`. Overall I suggest to change types to `numSlaves int, timeout time.Duration` so it reads better:

```
client.Wait(1, time.Minute)
```